app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,8 @@
 $baseFontSize: 12px;
 
-@import "bootstrap-variables";
 @import "bootstrap";
 @import "bootstrap-responsive";
+@import "bootstrap-variables";
```

Comment:
This file was empty before, but its first line does read `/* Fork specific bootstrap overrides go here. This is included before bootstrap. */`. Perhaps I should create a different file, `bootstrap-overrides` to be clearer.